### DISCUSSION TITLE
Cost Scaler, coef, Geo level

### DISCUSSION THREAD

**Discussion by wenwenzws** [google/lightweight_mmm#47] [Q&A]
Hi, 
Thanks for posting this package. Very excited about that!  I am running our package with real data. 
1. I found there is a piece that says cost scaler, "multiply=0.15". This impact the base contribution and roi a lot, from 0% to 80%. It looks quite weird sometimes that we don't have base contributions. I wonder why we set 0.15. Is this related to the prior?
2. How do you suggest if we want to calibrate the coef with the actual test result?
3. For GEO model, if we have 180+ dmas, it seems prior would have less effect? What do you suggest there? 
Thanks! 

Vera

**Comment by pabloduque0** [comment]
Hey @wenwenzws ! Thanks for opening this discussion!

1.  The `0.15` only applies to the dummy data and you will need to test what is the right scale factor for your case (or maybe none). Yes essentially you are setting the media priors (the scale of the the HalfNormal distribution) with your cost vector, therefore that prior will highly depend on your data, your extra features and other things that are related to your specific case.
2. If you either have results from experiments or prior knowledge about what percentage you media contribution looks like, you can use those to calibrate. 
3. For geo models you will have a hierarchical prior, so a national channel level prior (set by these values) will feed into a the geo-channel level prior. Although I would try to aggregate dmas if possible as 180+ geos might lead to very slow sampling (unless you have enough compute to throw at this, if yes, by all means).

**Reply by wenwenzws** [reply]
Thanks for your answer!
1. How do we specifically use these scalers to operate the prior or change the scale of half normal distribution on sigma?
2. do you suggest setting on the coefficient of the channels as prior, for eg. setting means to the test result?

**Reply by pabloduque0** [reply]
Sorry, I missed this one @wenwenzws .

1. Normally you would always use `divide_operation=jnp.mean` and then rely on `multiply_by` to either scale up or down the output values.
2. You can technically use the posterior of previous models as your priors for future models. Not sure if that is exactly what you are asking. Could you please clarify your question?
